docs/API.md

```diff
@@ -1,47 +1,4 @@
 # Component API
 
 * [Component props](APIProps.md)
-* [Component methods](APIMethods.md)
-
-## Props quick reference
```

Comment:
not exactly: the tools rely on package.json `"types"` key to tell them where to find an entry `.d.ts` file, and the `node_modules/@types` scope directory to list all the available external modules.